src/dotnet-structuring.library/Validator.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace dotnet_structuring.library
+{
+    public class Validator
+    {
+        public bool Number(string input)
+        {
+            var valid = false;
+
+            while (!valid)
+            {
+                valid = !string.IsNullOrWhiteSpace(input) &&
+                            input.All(c => c >= '0' && c <= '9');
```

Comment:
was tut das ?

Reply:
Die Validator.cs Klasse fällt weg, da ich einen anderen weg, um festzustellen ob sich nur zahlen in einem string befinden, gefunden habe.